express-api/package.json

```diff
@@ -38,10 +38,12 @@
     "pg": "8.12.0",
     "reflect-metadata": "0.2.1",
     "swagger-ui-express": "5.0.0",
+    "ts-node": "10.9.1",
+    "tsconfig-paths": "4.2.0",
     "typeorm": "0.3.20",
     "typeorm-naming-strategies": "4.1.0",
     "winston": "3.13.0",
-    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz",
+    "xlsx": "0.18.5",
```

Comment:
[Same comment as last time](https://github.com/bcgov/PIMS/pull/2521#discussion_r1676221873). Is there a way to keep this from coming up in the dependency reports?

Reply:
couldnt get the ignorePackages to work, instead included a blurb on the Jira ticket about xlsx